core/admin.py

```diff
@@ -12,6 +12,13 @@ class AccountAdmin(admin.ModelAdmin):
     actions = None
     list_display = ("title",)
 
+    def render_change_form(
+        self, request, context, add=False, change=False, form_url="", obj=None
+    ):
+        rcf = super().render_change_form(request, context, add, change, form_url, obj)
+        rcf.context_data["has_delete_permission"] = False
+        return rcf
```

Comment:
Could we use [has_delete_permission](https://docs.djangoproject.com/en/2.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.has_delete_permission) here? For example:

```python
class AccountAdmin(admin.ModelAdmin):

    def has_delete_permission(self, request, obj=None):
        return False
```

Reply:
Much simpler